src/components/SmartLabel.brs

```diff
@@ -147,13 +149,14 @@ function updateFonts(event = invalid)
   if event = invalid
     fontNames = m.fontFields
   else
-    fontNames = [event.getField()]
+    field = event.getField()
+    fontNames = [field.replace("Settings", "")]
   end if
 
-  for each name in fontNames
-    fontSettings = m.top[name + "Settings"]
```

Comment:
why change this to something less specific? Don't get this change. `fontSettings` is clearer than `settings` here.

Reply:
Updated